website/public/js/controllers/rootCtrl.js

```diff
@@ -184,23 +184,73 @@ habitrpg.controller("RootCtrl", ['$scope', '$rootScope', '$location', 'User', '$
           $rootScope.charts[id] = (history.length == 0) ? false : !$rootScope.charts[id];
           if (task && task._editing) task._editing = false;
       }
+
+      if($rootScope.charts[id]) {
```

Comment:
We've instituted eslint for the server code. We're about to do a rewrite of the client code, so we haven't bothered including it in the linting, since it will be replaced soon.